test/e2e/basic/sysctl_d_override.go

```diff
@@ -38,9 +38,7 @@ var _ = ginkgo.Describe("[basic][sysctl_d_override] Node Tuning Operator /etc/sy
 
 			if node != nil {
 				util.ExecAndLogCommand("oc", "label", "node", "--overwrite", node.Name, nodeLabelSysctlOverride+"-")
-			}
-			if pod != nil {
-				util.ExecAndLogCommand("oc", "exec", "-n", ntoconfig.WatchNamespace(), pod.Name, "--", "rm", sysctlFile)
+				util.ExecAndLogCommand("oc", "debug", fmt.Sprintf("no/%s", node.Name), "--", "rm", sysctlFile)
```

Comment:
here and following: could you please elaborate why `oc debug` exposes less than `oc exec`? It seems equivalente (or perhaps worse) than what we have currently

Reply:
Thank you for looking!  True, however, this is code in e2e tests.  The main idea is not to expose the entire `/host` in Tuned pods.  Happy to share internal links to customer questions why we are exposing the entire `/host` over Slack if you're interested.